pcsx2/gui/MainFrame.cpp

```diff
@@ -665,7 +665,7 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	menubar.Enable( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
+	menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
```

Comment:
@turtleli FYI, I did notice this change between the original function and my changes while testing.

The normal function is ```menubar.Enable``` - the Shutdown option is greyed out when you first launch the exe; but it doesn't get disabled when Shutdown is clicked .

When I changed the function to ```menubar.Check``` the Shutdown isn't greyed out when the exe is first launch; but it does get greyed out when Shutdown is clicked